tests/multipleBackend/objectPutCopyPart.js

```diff
@@ -158,6 +161,18 @@ describeSkipIfE2E('ObjectCopyPutPart API with multiple backends', () => {
         });
     });
 
+    it('should copy part to AWS based on mpu location', done => {
+        copyPutPart('mem', 'aws-test', null, 'localhost', uploadId => {
+            assert.strictEqual(ds.length, 2);
+            s3.abortMultipartUpload({ Bucket: 'multitester444',
```

Comment:
let's do a list parts on aws to check it's there.  why do we expect the ds.length to be 2 here?